extensions/functions_datetime.yaml

```diff
@@ -193,3 +221,45 @@ scalar_functions:
           - value: interval_year
           - value: interval_year
         return: boolean
+  -
+    name: "assume_timezone"
+    description: Convert local time to UTC using given local time's timezone.
+    impls:
+      - args:
+          - value: string
+          - value: timestamp
+        return: timestamp_tz
+  -
+    name: "strptime"
+    description: Parse string into timestamp/date/time using provided format,
+      see https://man7.org/linux/man-pages/man3/strptime.3.html for reference.
+    impls:
+      - args:
+          - value: string
+          - value: string
```

Comment:
Which of these is the format and which is the timestamp? In general, please name the arguments if it's not abundantly obvious what they represent (just add a `name` key to the argument descriptor object).

Reply:
Added names.